python-plugin/src/main/java/fr/greencodeinitiative/python/checks/DetectUnoptimizedImageFormat.java

```diff
@@ -0,0 +1,41 @@
+package fr.greencodeinitiative.python.checks;
+
+import org.sonar.check.Priority;
+import org.sonar.check.Rule;
+import org.sonar.plugins.python.api.PythonSubscriptionCheck;
+import org.sonar.plugins.python.api.SubscriptionContext;
+import org.sonar.plugins.python.api.tree.StringLiteral;
+import org.sonar.plugins.python.api.tree.Tree;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Rule(
+        key = DetectUnoptimizedImageFormat.RULE_KEY,
+        name = DetectUnoptimizedImageFormat.MESSAGERULE,
+        description = DetectUnoptimizedImageFormat.MESSAGEERROR,
+        priority = Priority.MINOR,
+        tags = {"bug", "eco-design", "ecocode"})
```

Comment:
maybe not a "bug" but a "performance" tag ? check if other tags are interesting, please.